### ISSUE TITLE
unescaped html in doc/web/howto/web-in-60/handling-posts.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4336]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4336|
|Type|enhancement|
|Created|2010-03-01 22:58:11Z|
|Branch|https://github.com/twisted/twisted/tree/web-html-escape-4336|

in doc/web/howto/web-in-60/handling-posts.xhtml there is some markup like:

```
<pre class="python">
class FormPage(Resource):
    def render_GET(self, request):
        return '<html><body><form method="POST"><input name="the-field" type="text" /></form></body></html>'
</pre>
```

In the rest of this documentation, the html tags inside the <pre> tags is escaped, like this:

```
<pre class="python">
class FormPage(Resource):
    def render_GET(self, request):
        return '&lt;html&gt;&lt;body&gt;&lt;form method="POST"&gt;&lt;input name="the-field" type="text" /&gt;&lt;/form&gt;&lt;/body&gt;&lt;/html&gt;'
</pre>
```

This page should be fixed to match.

<details><summary>Searchable metadata</summary>

```
trac-id__4336 4336
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__branches_web_html_escape_4336 branches/web-html-escape-4336
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1267484291000000 1267484291000000
changetime__1268507147000000 1268507147000000
version__None None
owner__jesstess jesstess
cc__khorn cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @jyknight|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess commented|
|-|-|

(In [28667]) Branching to 'web-html-escape-4336'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess commented|
|-|-|

(In [28672]) Escape html tags inside pre blocks in the web in 60 series of examples.

refs [#4336](https://github.com/twisted/twisted/issues/4336)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

I had a quick look at the changes.  Mostly seems good (hard to avoid having one's eyes glaze over though).  I did notice one mistake though, it looks like the indentation in `FormPage.render_POST`  in `handling-posts.xhtml` got messed up.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess commented|
|-|-|

(In [28674]) Lost an indent in handling-posts.

refs [#4336](https://github.com/twisted/twisted/issues/4336)

**Comment by twisted-trac** [comment]
|<img alt="sirgolan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_8"></a>|sirgolan set owner to @jesstess|
|-|-|

Looks good to me. Ran a doc build that passed as well:

http://buildbot.twistedmatrix.com/builders/documentation/builds/294

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_9"></a>|@jesstess set status to `closed`|
|-|-|

Merge web-html-escape-4336

Author: jesstess
Reviewers: exarkun, sirgolan
Fixes: [#4336](https://github.com/twisted/twisted/issues/4336)

HTML tags inside <pre> tags in the code snippets are now escaped.
